xml_converter/generators/cpp_templates/class_template.cpp

```diff
@@ -214,17 +237,26 @@ void {{cpp_class}}::parse_protobuf(waypoint::{{cpp_class}} proto_{{cpp_class_hea
         this->{{attribute_variable.attribute_name}} = from_proto_{{attribute_variable.class_name}}(proto_{{cpp_class_header}}.{{attribute_variable.protobuf_field}}());
         this->{{attribute_variable.attribute_name}}_is_set = true;
     }
-    {% elif (attribute_variable.attribute_type in ["MultiflagValue", "CompoundValue", "Custom"]) and attribute_variable.is_child == false%}
+    {% elif (attribute_variable.attribute_type in ["MultiflagValue", "CompoundValue", "Custom"]) and attribute_variable.compound_name == None%}
+    {% if (attribute_variable.protobuf_field == "rgba")%}
```

Comment:
Don't branch on arbitrary documentation strings. This would mean that if someone wanted to change the documentation they might also need to change this template file too.

Reply:
Understood. Reverting this.